tests/unit/date/core.js

```diff
@@ -1,19 +1,21 @@
 define( [
 	"jquery",
-	"globalize",
+	"./helper",
 	"date"
-], function( $, Globalize ) {
+], function( $, testHelper ) {
 
 module( "date: core" );
 
+var attributes = testHelper.getAttributes( "en" );
```

Comment:
We currently do not use any other attributes object. Where needed, we change Globalize locale directly.